### PR TITLE
Fix SpotBugs warning

### PR DESCRIPTION
**Pull request by MarkEWaite**
## Fix SpotBugs warning

- Resolve spotbugs warning with an assignment

## Checklist

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- ~~Ensure you have provided tests - that demonstrates feature works or fixes the issue~~

### CODE REVIEW

**Comment by MarkEWaite** [comment]
Closing in favor of #23 and its change to Eclipse Temurin

**Comment by basil** [comment]
There were some valuable SpotBugs cleanups in this PR, so maybe it doesn't need to be closed? Can just be rebase on top of #23 and the SpotBugs fixes retained.

File: pom.xml [hunk 1]
```diff
@@ -56,6 +56,7 @@
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
+
```
Comment (basil) [inline pom.xml:59]:
```suggestion
```

File: pom.xml [hunk 2]
```diff
@@ -70,6 +71,12 @@
             <artifactId>wiremock-jre8-standalone</artifactId>
             <version>2.33.2</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-api</artifactId>
+                </exclusion>
+            </exclusions>
```
Comment (basil) [inline pom.xml:79]:
```suggestion
```

**Review by basil — APPROVED** [review]
Thanks for the PR!

**Pull request merged**